cmake/Buildrobots-configuration.cmake

```diff
@@ -8,11 +8,20 @@ include(FindOrBuildPackage)
 find_or_build_package(YARP QUIET)
 find_or_build_package(ICUBcontrib QUIET)
 
+# This custom code is present to make sure that:
+# * On iCub Head images, the behavior is to install only those specific robot files
+# * On other systems (i.e. developers laptop) all the robots configuration files are installed
+set(robots-configuration_CMAKE_ARGS "")
+if("$ENV{YARP_ROBOT_NAME}" STREQUAL "")
+    list(APPEND robots-configuration_CMAKE_ARGS "-DINSTALL_ALL_ROBOTS:BOOL=ON")
+endif()
+
 
 ycm_ep_helper(robots-configuration TYPE GIT
                                    STYLE GITHUB
                                    REPOSITORY robotology/robots-configuration.git
                                    DEPENDS YARP
                                            ICUBcontrib
                                    COMPONENT iCub
-                                   FOLDER robotology)
+                                   FOLDER robotology
+                                   CMAKE_ARGS ${configuration_CMAKE_ARGS})
```

Comment:
Should this be `${robots-configuration_CMAKE_ARGS}`?

Reply:
Good catch, fixed by https://github.com/robotology/robotology-superbuild/pull/183/commits/40000f0ffcbd8e432a1a6cf6f2396e4d5f49ce75 .